java/pending_code_changes.txt

```diff
@@ -1 +1,8 @@
+Build changes:
+-- OSGi support added to Manifest information when building jar (#1300)
 
+Code changes:
+-- Formatting, naming (LOGGER -> logger) and comment tweaks to follow style
+guide
+-- Removal of unneeded canBeGeocoded method in the
+PhoneNumberToTimeZonesMapper.java, using phoneNumberUtil instead
```

Comment:
I think we should mention JS possible lengths.

IMO we don't need to mention the internal changes like style fixes and private method removal.